junebug/api.py

```diff
@@ -51,6 +52,17 @@ def setup(self, redis=None, message_sender=None):
         self.outbounds = OutboundMessageStore(
             self.redis, self.config.outbound_message_ttl)
 
+        yield self._start_stored_channels()
+
+    @inlineCallbacks
+    def _start_stored_channels(self):
+        ids = yield Channel.get_all(self.redis)
+        for id in ids:
+            properties = yield self.redis.get('%s:properties' % id)
+            properties = json.loads(properties)
+            channel = Channel(self.redis, self.config, properties, id)
+            yield channel.start(self.service)
```

Comment:
Would it maybe be worth encapsulating some of this in `Channel`? Maybe a classmethod that does something like `Channel.get_all_channels(self.redis, self.config)` that returns a list of `Channel` instances?